Tools/Gulp/gulpfile.js

```diff
@@ -77,6 +78,7 @@ gulp.task("buildNoWorker", ["shaders"], function () {
     )
     .pipe(concat(config.build.minNoWorkerFilename))
     .pipe(cleants())
+    .pipe(addModuleExports("BABYLON"))
```

Comment:
Why not just appending a file named footer.js?(I'm sure I missed something:))Leave a line note